onnxruntime/test/providers/qnn/qnn_test_utils.h

```diff
@@ -75,6 +72,18 @@ inline QuantParams<QType> GetDataQuantParams(gsl::span<const float> data) {
   return QuantParams<QType>::Compute(min_val, max_val);
 }
 
+/**
+ * Returns a float vector with data in the specified range. Uses linear interpolation to fill the elements in the array
+ * and ensures that min_val, 0.0f, and max_val are all included.
+ * TODO(adrianlizarraga): Should use this instead of random *float* test inputs for test repeatability/stability!
+ *
+ * \param min_val The minimum value.
+ * \param max_val The maximum value.
+ * \param num_elems The number of elements in the result.
+ * \return A vector of floats with elements set to values in the specified range.
+ */
+std::vector<float> GetFloatDataInRange(float min_val, float max_val, size_t num_elems);
```

Comment:
I'll try to use this for test that use random floating-point input in the next PR that adds more unit tests. I want to keep this one small.